### ISSUE TITLE
Has error initiating the Toolbar object in the function

### ISSUE THREAD

**Issue by lancevalour** [CosmicMind/Material#363]
I can't initiate Toolbar object with default constructor in the function. See below:
![image](https://cloud.githubusercontent.com/assets/7004418/15056661/afbef6ce-12e0-11e6-97bb-36937480dd80.png)

However if I initiate it outside the function in the class, there is no error:
![image](https://cloud.githubusercontent.com/assets/7004418/15056728/2ae943e0-12e1-11e6-9f3e-f19ac88bb34c.png)

**Comment by mohpor** [comment]
try:

```swift
mainToolBar = Toolbar(frame: CGRect.zero)
```

(Why? because of the init chaining it somehow loses the sight of which initializer to user)

**Comment by lancevalour** [comment]
yes, that works. I also try:

```
 mainToolBar = Toolbar(titleLabel: titleLabel,  leftControls: [menuButton], rightControls: [searchButton])

```
and it works too. Seems empty default constructor has some problems.

**Comment by danieldahan** [comment]
I fixed this in 1.40.1, which I will be pushing within the next 30 minutes.
